packages/synapse-react-client/src/utils/functions/TableColumnSchemaUtils.ts

```diff
@@ -47,23 +74,34 @@ export async function createTableUpdateTransactionRequest(
     .list
 
   // now that all columns have been created, figure out the column changes (create, update, and no-op)
-  // keep track of column ids to figure out what columns were deleted.
   const changes: ColumnChange[] = []
-  const columnIds: Set<string | null> = new Set()
+  const columnIdsThatWereChangedOrAdded: Set<string> = new Set()
```

Comment:
Rename to `idsThatShouldNotBeDeletedFromSchema`,

Consider adding comments about _why_ we add each value to this set when we do so.